doc/data/messages/d/deprecated-decorator/good.py

```diff
@@ -1 +1,3 @@
-# This is a placeholder for correct code for this message.
+class MyClass:
+    def my_method(cls):
```

Comment:
We need to use the replacement of ``@abc.abstractclassmethod``

Reply:
It is classmethod